es/reference/install.rst

```diff
@@ -70,26 +61,78 @@ Compilando la extensión:
 
 .. code-block:: bash
 
-    git clone git://github.com/phalcon/cphalcon.git
+    git clone --depth=1 git://github.com/phalcon/cphalcon.git
     cd cphalcon/build
     sudo ./install
 
-Añadiendo la extensión a php.ini
+Añadiendo la extensión a php.ini:
 
 .. code-block:: bash
 
+    # Suse: Add this line in your php.ini
+    extension=phalcon.so
+
+    # Centos/RedHat/Fedora: Add a file called phalcon.ini in /etc/php.d/ with this content:
+    extension=phalcon.so
+
+    # Ubuntu/Debian: Add a file called 30-phalcon.ini in /etc/php5/conf.d/ with this content:
+    extension=phalcon.so
```

Comment:
May be?

**Ubuntu/Debian with Apache**
Add a file called `30-phalcon.ini` in `/etc/php5/apache2/conf.d/` with this content:
```ini
extension=phalcon.so
```
**Ubuntu/Debian with PHP-FPM**
Add a file called `30-phalcon.ini` in `/etc/php5/fpm/conf.d/` with this content:
```ini
extension=phalcon.so
```
**Ubuntu/Debian with PHP-CLI**
Add a file called `30-phalcon.ini` in `/etc/php5/cli/conf.d/` with this content:
```ini
extension=phalcon.so
```